### PR TITLE
enhancement of expiry shorcut buttons

### PR DESCRIPTION
**Pull request by belljohan3**

### CODE REVIEW

**Comment by ivanslabbert** [comment]
@belljohan3 can you mark the selected button (first selected) so that the user will know his selection is active?

File: cms/src/components/content/EditContentForm.vue [hunk 1]
```diff
@@ -760,3 +809,10 @@ const checkIfDirty = () => {
         <ConfirmBeforeLeavingModal :isDirty="isDirty" />
     </form>
 </template>
+
+<!-- <style scoped>
+.active {
+    background-color: #000000; /* or any other color to indicate selection */
+    color: white;
+}
+</style> -->
```
Comment (ivanslabbert) [inline cms/src/components/content/EditContentForm.vue:818]: Do we need to keep this?

**Review by ivanslabbert — APPROVED** [review]
Thanks Johan, nice work. Just check my comment, and merge when ready.

File: cms/src/components/content/EditContentForm.vue (hunk 1 above)
Comment (belljohan3) [reply cms/src/components/content/EditContentForm.vue]: No ! I will delete it

**Pull request merged**